lib/index.js

```diff
@@ -76,16 +76,20 @@ const tmpDir = () => path.join(os.tmpdir(), TMP_DIR.replace(TEMPLATE_INTERPOLATE
 
 // exports
 
+export function registerCleanup(fn) {
+  tempXCallbacks.push(fn);
+  return function manualCleanupOne() {
+    fn();
+    tempXCallbacks = tempXCallbacks.filter((callback) => callback !== fn);
+  }
+}
+
```

Comment:
Why export `registerCleanup`? Doesn't seem necessary.

Reply:
Good catch! Muscle memory I guess :sweat_smile: I've removed the `export` declaration.